addons/calendar/models/mail_message.py

```diff
@@ -26,6 +26,6 @@ def search(self, args, offset=0, limit=0, order=None, count=False):
     def _find_allowed_model_wise(self, doc_model, doc_dict):
         if doc_model == 'calendar.event':
             order = self._context.get('order', self._order)
-            for virtual_id in self.env[doc_model].get_recurrent_ids(doc_dict.keys(), [], order=order):
```

Comment:
I think this fix is wrong. It should be self.env[].browse(dict.keys).get_recurrent_ids([]) . I will fix it and see if backporting is required.